dev/ProgressRing/ProgressRingLoading.cpp

```diff
@@ -0,0 +1,362 @@
+﻿// This file was auto-generated then been modified to update animation visuals with inherited visual properties
+
+#include "pch.h"
+#include "common.h"
+#include "ProgressRingLoading.h"
+#include "WindowsNumerics.h"
+
+static bool IsRuntimeCompatible()
+{
+    if (!winrt::Windows::Foundation::Metadata::ApiInformation::IsTypePresent(L"Windows.UI.Composition.CompositionGeometricClip"))
```

Comment:
>CompositionGeometricClip [](start = 102, length = 24)

I can tell from this line that you generated the code from an older version of Lottie Viewer. Please upgrade to the latest version. That's going to be a pain seeing as you have to translate CX back to winrtcpp, but I can probably automate a bunch of that in LottieGen with some guidance from you, so let's do that rather than checking in content generated from older versions.

Reply:
@simeoncran Thank you for catching that. Yes, the manual translation from CX to winrtcpp is not idea. However, it looks like there is already work being done in this area. See https://github.com/windows-toolkit/Lottie-Windows/issues/120. @clarkezone - Is this currently in progress ?